src/main/java/study/domain/racingGame/CarCondition.java

```diff
@@ -0,0 +1,21 @@
+package study.domain.racingGame;
+
+import java.util.Random;
+
+public class CarCondition {
+  private final int MIN_NUM = 4;
+  private final int MAX_NUM = 9;
```

Comment:
상수는 다음과 같이 static 을 선언해 주면 어떨까요? 😄 
```suggestion
public class CarCondition {
  private static final int MIN_NUM = 4;
  private static final int MAX_NUM = 9;
```

Reply:
넵 값이 고정되어있어서 객체를 생성할 때마다 메모리를 사용하는 것는 불필요한 내용이네요 static으로 수정하도록 하겠습니다.